external/src/main/resources/application.properties

```diff
@@ -107,4 +107,8 @@ quarkus.log.cloudwatch.log-group=policiesLogGroup
 quarkus.log.cloudwatch.log-stream-name=policies-engine
 quarkus.log.cloudwatch.level=INFO
 quarkus.log.cloudwatch.access-key-id=placeholder
-quarkus.log.cloudwatch.access-key-secret=placeholder
\ No newline at end of file
+quarkus.log.cloudwatch.access-key-secret=placeholder
+
+# This is a new feature introduced in Quarkus 1.13 and used by the dev UI.
+# It is causing failures when the app is started with quarkus:dev because some parts of our code are not compliant with the CDI spec.
+quarkus.arc.dev-mode.monitoring-enabled=false
```

Comment:
@josejulio This fixed the plugin registration issue.

It also fixed the reason why I migrated metrics from MicroProfile Metrics to Micrometer. Since the removal of MP Metrics caused a weird Infinispan error logged at DEBUG level, I reverted the migration and we're back the initial metrics situation.

Could you please test policies one more time? I hope it will be the last one for this bump!